src/tokens/pancakeswap-eth-mm.json

```diff
@@ -134,5 +134,13 @@
     "chainId": 1,
     "decimals": 18,
     "logoURI": "https://tokens.pancakeswap.finance/images/eth/0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85.png"
+  },
+  {
```

Comment:
we should add to eth-default instead of eth-mm

Reply:
i asked momota on this he said to both, but can remove if u think its better

Reply:
`src/tokens` also need add `pancakeswap-eth-default.json`

Reply:
I don't think mm support all the tokens. We better check if we have the token whitelisted on be before we add to the tokenlist

Reply:
this is the mm list not the mini program list